### ISSUE TITLE
"null" into "hiddenApps" tab (indiegala_giveaways.js)

### ISSUE THREAD

**Issue by TineArconn** [RedSparr0w/IndieGala-Helper#1]
Hi !

I don't know how, but I ended up with a null value in the variable hiddenApps (file indiegala_giveaways.js), which poses a problem line 26: app.replace() crash the plugin. Maybe it should add a check ?.

(Sorry for my english)

**Comment by RedSparr0w** [comment]
Just uploaded a new update, would you mind checking if this has fixed the problem [=

v1.6.1

edit: are you using chrome or firefox?

**Comment by TineArconn** [comment]
I use Chromium / Chrome.

I can't reproduce the bug. I think it comes from Indiegala behavior : sometimes, names of giveaways may be different from images or have names like "?????????: ??????? ?????????????", I guess I had to have a name "null" once.

**Comment by RedSparr0w** [comment]
there was a problem with `Worms Reloaded: The "Pre-order Forts and Hats" DLC Pack` due to the quote marks in the name not sure what would have caused a null name though.

Have you had a chance to test out the latest version and see if that has fixed the error?

**Comment by TineArconn** [comment]
I tried to reproduce the bug with the interface, but I have not found.

So, I added a string "null" value to the value of hiddenApps localStorage manually (via F12 / Resources / Local Storage).

We can see that the string "null" value is visible in the recovered table (that's what I had when I reported the problem), but the plugin does not crash by cons.

![null](https://cloud.githubusercontent.com/assets/7766377/16731272/7233b90e-4777-11e6-908b-0ead0f2f4aa1.png)

PS : using 1.6.2 version
